.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "test/googletest"]
```

Comment:
Sollte man gtest nicht auch wie nlohmann::json als externe dependency mit systemweiter Installation aufnehmen?

Reply:
https://cliutils.gitlab.io/modern-cmake/chapters/testing/googletest.html

> Downloading a copy for each project is the recommended way to use GoogleTest (so much so, in fact, that they have disabled the automatic CMake install target), so this respects that design decision.

Vielleicht sollten/könnten wir das in Zukunft sogar so machen dass das von cmake runtergeladen wird, statt als submodule:

https://github.com/google/googletest/blob/master/googletest/README.md#incorporating-into-an-existing-cmake-project

Reply:
#9